### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smaticka**
## Description

Adding instrument compatibility for the converter.

### CODE REVIEW

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @smaticka!

File: src/ncar-bufr2nc-fortran/README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # bufr2nc_fortran.x
 
+## Current instruments supported:
+* iasi
+* amsua
+* mhs
+* gpsro is supported but the recommended converter to use is `src/gnssro/gnssro_bufr2ioda.f90`
```
Comment (BenjaminRuston) [inline src/ncar-bufr2nc-fortran/README.md:7]:
```suggestion
* gpsro is supported but the recommended converter to use is `src/gnssro/gnssro_bufr2ioda.f90`
* prepBUFR which outputs ascan, satwind, aircraft and radiosonde
```

**Comment by BenjaminRuston** [comment]
@smaticka need to double check for prepBUFR the files output there are the files it outputs and the ones we've tested and used through skylab

**Comment by smaticka** [comment]
> @smaticka need to double check for prepBUFR the files output there are the files it outputs and the ones we've tested and used through skylab

@BenjaminRuston, For double checking, do you mean to run the `bufr2nc_fortran.x` converter on each of the output files (ascan, satwind, aircraft and radiosonde) to make sure it works for those?

File: src/ncar-bufr2nc-fortran/README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+* prepBUFR which outputs ascan, satwind, aircraft and radiosonde
```
Comment (BenjaminRuston) [inline src/ncar-bufr2nc-fortran/README.md:8]:
```suggestion
* prepBUFR which outputs ascan, satwind, aircraft, surface observations and radiosonde
```

**Review by BenjaminRuston — APPROVED** [review]
this is good for now we may want to update as we solidify what we are still using or recommend to use out of these converters

**Pull request merged**